Scripts/lint.sh

```
@@ -0,0 +1,101 @@
+
+if [ "$LINT_MODE" = "NONE" ]; then
+	exit
+elif [ "$LINT_MODE" = "STRICT" ]; then
+	SWIFTFORMAT_LINT_OPTIONS="--strict"
+	SWIFTLINT_OPTIONS="--strict"
+else
+	SWIFTFORMAT_LINT_OPTIONS=""
+	SWIFTLINT_OPTIONS=""
+fi
+
+pushd $PACKAGE_DIR
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)